src/Activities/Option/ActivityOption.cs

```diff
@@ -17,5 +17,10 @@ internal class ActivityOption
 		/// Disable ActivityEventSender so Activity metric is only sent via ActivityMetricsSender
 		/// </summary>
 		public bool ActivityEventSenderEnabled { get; set; } = true;
+
+		/// <summary>
+		/// Sets each activities parent name as a dimension value.
+		/// </summary>
+		public bool SetParentNameAsDimension { get; set; } = false;
```

Comment:
```suggestion
		public bool SetParentNameAsDimensionEnabled { get; set; } = false;
```It is normally have "Enabled" suffix for boolean option